config/locales/en/documents/show.yml

```diff
@@ -97,3 +97,6 @@ en:
         unschedule_error:
           title: Something has gone wrong
           description_govspeak: Something went wrong when stopping the scheduling of this document. Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
+        backdate_error:
+          title: Something has gone wrong
+          description_govspeak: Something went wrong when changing the backdate of this document. Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
```

Comment:
We shouldn't need this as this error can't occur

Reply:
Removed